cli-messaging/src/main/java/com/salesforce/messaging/Message.java

```diff
@@ -0,0 +1,47 @@
+package com.salesforce.messaging;
```

Comment:
Basically a copy of `pmd-cataloger/src/main/java/sfdc/sfdx/messaging/Message.java`.